src/coremidi.cs

```diff
@@ -216,17 +214,5 @@ interface MidiCISession
 
 		[NullAllowed, Export ("profileChangedCallback", ArgumentSemantic.Assign)]
 		MidiCIProfileChangedHandler ProfileChangedCallback { get; set; }
-
-		[Export ("hasProperty:onChannel:responseHandler:")]
```

Comment:
These show up in the headers.

Reply:
yees submissions complain.

Reply:
Only ref I can find is when it was added. But nothing else.